build.sh

```diff
@@ -343,7 +343,7 @@ log_action_end_msg $?
 
 # configure appropriate init system
```

Comment:
@CaseyLabs after looking (again) at the mess that is https://unix.stackexchange.com/questions/18209/detect-init-system-using-the-shell, I am thinking it may be best to just default to systemd, since it's 2021 and if anything is still relying on upstart, we don't want to support it.

Certainly Debian 9/10 and Ubuntu 16+ are all on systemd.

So I propose refactoring this block:

```
# configure appropriate init system
log_action_begin_msg "configuring init system"
if [[ `/lib/systemd/systemd --version` =~ upstart ]]; then
    sudo cp ${CWD}/init/*.conf /etc/init/ &>> ${CWD}/netflix-proxy.log\
      && sudo $(which sed) -i'' "s#{{CWD}}#${CWD}#g" /etc/init/netflix-proxy-admin.conf &>> ${CWD}/netflix-proxy.log\
      && sudo service netflix-proxy-admin restart &>> ${CWD}/netflix-proxy.log
fi

if [[ `systemctl` =~ -\.mount ]]; then
      sudo cp ${CWD}/init/*.service /lib/systemd/system/ &>> ${CWD}/netflix-proxy.log\
        && sudo $(which sed) -i'' "s#{{CWD}}#${CWD}#g" /lib/systemd/system/netflix-proxy-admin.service &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl daemon-reload &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl enable netflix-proxy-admin &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl enable systemd-networkd &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl enable systemd-networkd-wait-online &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl restart netflix-proxy-admin &>> ${CWD}/netflix-proxy.log
fi
log_action_end_msg $?
```

To something simple like:

```
# configure init system
log_action_begin_msg "configuring init system"
if [[ `/lib/systemd/systemd --version` =~ systemd ]]; then
      sudo cp ${CWD}/init/*.service /lib/systemd/system/ &>> ${CWD}/netflix-proxy.log\
        && sudo $(which sed) -i'' "s#{{CWD}}#${CWD}#g" /lib/systemd/system/netflix-proxy-admin.service &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl daemon-reload &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl enable netflix-proxy-admin &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl enable systemd-networkd &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl enable systemd-networkd-wait-online &>> ${CWD}/netflix-proxy.log\
        && sudo systemctl restart netflix-proxy-admin &>> ${CWD}/netflix-proxy.log
else
    echo 'unsupported init system' && exit 1
fi
log_action_end_msg $?
```

and removing all the upstart related assets (init/*.conf) from the repository...